Swabbr.Api/Controllers/AuthenticationController.cs

```diff
@@ -232,24 +233,14 @@ public async Task<IActionResult> ChangePasswordAsync([FromBody] UserChangePasswo
         }
 
         /// <summary>
-        /// Deauthorizes the authenticated user.
+        ///     Deauthorizes the authenticated user.
         /// </summary>
-        /// <returns><see cref="NoContentResult"/></returns>
-        [Authorize]
         [HttpPost("logout")]
         [ProducesResponseType((int)HttpStatusCode.NoContent)]
         public async Task<IActionResult> LogoutAsync()
         {
             try
             {
-                using (var scope = new TransactionScope(TransactionScopeAsyncFlowOption.Enabled))
-                {
-                    // Unregister device
-                    var user = await _userManager.GetUserAsync(User).ConfigureAwait(false);
-                    await _deviceRegistrationService.UnregisterAsync(user.Id).ConfigureAwait(false);
```

Comment:
Logging out meaned that we would unsubscribe from push notifications. This is why we received little to no notifications during the testing phase.